frontend/routes/package/source.tsx

```diff
@@ -62,19 +62,23 @@ export default function PackagePage(
       />
 
       <div class="space-y-3 pt-3">
-        <div class="pl-2">
+        <div class="flex flex-row gap-1 items-center pt-1 pl-3">
           {data.sourcePath.split("/").filter((part, i) =>
             !(part === "" && i !== 0)
           ).map((part, i, arr) => {
             if (part === "") {
               // @ts-ignore ok
-              part = <span class="italic">root</span>;
+              part = (
+                <span class="text-lg font-semibold">
+                  Package root
```

Comment:
Root seemed like an ambiguous word, open to other suggestions but went for something a bit more verbose that alludes to what files we're looking at here.